src/main/resources/fabric.mod.json

```diff
@@ -4,7 +4,7 @@
   "version": "${version}",
 
   "name": "Incubus Core",
-  "description": "Bring your own lube.",
+  "description": "Inaba of the cum.",
```

Comment:
Inaba?

Reply:
Funny reference to multiple things, mainly touhou.